### ISSUE TITLE
Copying portable install from C drive to E drive hangs on embeddings

### ISSUE THREAD

**Issue by Amitosh3498275** [LykosAI/StabilityMatrix#1068]
### Package

"embeddings" in the package folder hangs while copying.

### When did the issue occur?

Other

### What GPU / hardware type are you using?

NVIDIA RTX 3050 Laptop GPU

### What happened?

Okay, so i installed the portable version and while copying the "data" and .exe file to another drive, the process hangs indefinitely.
When i located the file causing the issue, it gave me this error on clicking it.
![image](https://github.com/user-attachments/assets/9231d565-7f91-4b8c-947b-a820f901eb74)

### Version

v2.13.0

### What Operating System are you using?

Windows

**Comment by illlustr** [comment]
I face the same problem before. it solved by enabling `remove shared checkpoints directory symbolic link on shutdown` in settings and disable all sharing in `Packages`

**Comment by Amitosh3498275** [comment]
Thank you. I'll try this one.

**Comment by mohnjiles** [comment]
> I face the same problem before. it solved by enabling `remove shared checkpoints directory symbolic link on shutdown` in settings and disable all sharing in `Packages`

Yep this is the solution, unfortunately it is an issue with File Explorer when dealing with symlink/junction directories. Alternative file transfer software may not have this issue. We should probably add another easy checkbox to disable the output sharing links on shutdown also.
